src/pages/[platform]/build-a-backend/auth/customize-auth-lifecycle/custom-auth-flows/index.mdx

```diff
@@ -946,7 +824,7 @@ user has already signed in and a valid session is active. You must first call
 </Callout>
 ## Confirm sign in with custom challenge
 
-Get the custom challenge (`1234` in this case) from the user and pass it to the `confirmSignin()` api.
+Get the custom challenge from the user and pass it to the `confirmSignin()` api.
```

Comment:
How to get it from the user? They will submit the answer to the challenge through a UI?

Reply:
Yes and the functions below has a parameter for it.

Reply:
Can you update the wording for that i.e. instead of get the ..etc perhaps use something like "Present a UI for the user to submit the answer to the custom challenge and use it ...etc"